lib/wor/paginate/config.rb

```diff
@@ -23,10 +34,30 @@ module Config
         end
       end
 
+      def add_adapter(adapter)
+        @adapters << adapter
+      end
+
+      def remove_adapter(adapter)
+        @adapters.delete(adapter) if @adapters.include? adapter
```

Comment:
Is this check necessary?

Reply:
You're totally right, it's not necessary 😅